torchinductor/triton_ops/matmul.py

```diff
@@ -58,7 +58,7 @@ def _kernel(
         else:
             a = tl.load(A, mask=rk[None, :] < k, other=0.0)
             b = tl.load(B, mask=rk[:, None] < k, other=0.0)
-        acc += tl.dot(a, b, allow_tf32=False)
+        acc += tl.dot(a, b)
```

Comment:
can you add allow_tf32 argument to the kernel, it should be set by querying current pytorch global precision setting, so that comparisons are always fair? 
Same should be done for template that's used to generate matmul/conv.

Reply:
Sure. Thanks for the suggestions. I will do that in later PR